<?xml version="1.0"?>
<launch>

  <group ns="optical_flow_group">

<!--
  <rosparam param="camera_calibration">[1000, 0, 600, 0, 1000, 540, 0, 0, 1]</rosparam>
-->
<!-- PPT -->
<!--
  <rosparam param="camera_calibration">[1000.0, 0.0, 960, 0.0, 1000.0, 540, 0.0, 0.0, 1.0]</rosparam>
-->
<!--
  <rosparam param="camera_calibration">[1.0, 0.0, 960, 0.0, 1.0, 540, 0.0, 0.0, 1.0]</rosparam>
-->
<!--
  <rosparam param="camera_calibration">[780, 0, 256, 0, 780, 256, 0, 0, 1]</rosparam> 
-->
<!-- Off1 1 - primary calibration -->
  <rosparam param="camera_calibration">[769.030, 0.0, 512.0, 0.0, 769.030, 512.0, 0.0, 0.0, 1.0]</rosparam> 
<!-- Off1 2 - alternative calibrations -->
<!--
  <rosparam param="camera_calibration">[1056.1, 0.0, 512.0, 0.0, 950.50, 512.0, 0.0, 0.0, 1.0]</rosparam> 
-->
<!--
  <rosparam param="camera_calibration">[512.0, 0.0, 512.0, 0.0, 512.0, 512.0, 0.0, 0.0, 1.0]</rosparam> 
-->
<!--
  <rosparam param="camera_calibration">[10, 0.0, 512.0, 0.0, 10, 512.0, 0.0, 0.0, 1.0]</rosparam> 
-->
<!-- Off3 calibration -->
<!--
<rosparam param="camera_calibration">[512.0, 0.0, 512.0, 0.0, 512.0, 512.0, 0.0, 0.0, 1.0]</rosparam>
-->
<!-- iphone 1-->
  <rosparam param="camera_calibration">[570, 0, 160, 0, 570, 284, 0, 0, 1]</rosparam> 
<!--
  <rosparam param="camera_calibration">[1, 0, 160, 0, 1, 284, 0, 0, 1]</rosparam> 
-->
<!-- iphone 2-->
<!--
  <rosparam param="camera_calibration">[570, 0, 960, 0, 570, 540, 0, 0, 1]</rosparam> 
-->
<!--
  <rosparam param="camera_calibration">[570, 0, 160, 0, 570, 284, 0, 0, 1]</rosparam> 
-->
<!--
  <param name="video_source" value="/home/developer/Desktop/simulation_datasets/black/" />
-->
<!--
  <param name="truth_source" value="/home/developer/Desktop/2011_09_26/2011_09_26_drive_0009_sync/oxts/data/" />
-->
  <!--
  this is probably an unnecessary param
  <param name="truth_source" value="/home/developer/Desktop/2011_09_26/2011_09_26_drive_0009_sync/oxts/data/" />
  -->
  
<!--
  <param name="num_images" value="447" />
-->
<!--
  <param name="num_images" value="153" />
-->
  <param name="num_trials" value="1" />
<!--
  <param name="hertz" value="1" />
-->
  <param name="hertz" value="10" />
  
  <param name="poly_limit" value="2" />
	  
<!--
    <node pkg="optical_flow" name="image_pub" type="image_publisher.py" output="screen"/>
    <node pkg="optical_flow" name="image_sub" type="lk_track_ros.py" output="screen"/>
-->
    <node pkg="testing_and_analyzing" name="image_sub" type="readin_csv_fp.py" output="screen"/>
    <node pkg="optical_flow" name="velocity_estimator" type="velest.py" output="screen" >        
        <param name="NormalizeMode" type="int" value="2"/> <!-- 0=scaled with vest depth, 1=Magnitude of 1.0, 2=scaled with quest and vest depth  --> 
        <param name="using_SVD" type="bool" value="True"/> 
        <param name="using_BigC" type="bool" value="False"/> 
        <param name="NegateSol" type="bool" value="True"/> 
    </node>  
  </group>

</launch>


